templates/system.json

```diff
@@ -35,6 +35,12 @@
       "payload_property": "data",
       "url": "{{ properties.datatype }}?{{ properties.param }}={{ entity.gid }}"
     },
+    "teams_list": {
```

Comment:
This is just a copy of the tasks_list operation. Should we merge them with a more generic name?

Reply:
Yeah, if it's the same we can just update to a more generic name. I wasn't sure if any other datatypes would use this